### PR TITLE
fix: empty slice found error

### PR DESCRIPTION
**Pull request by dirablue**
fixed the error occurred when savePolicy is called.

```
github.com/casbin/gorm-adapter/v3@v3.7.1/adapter.go:580 empty slice found
[3.402ms] [rows:0] INSERT INTO `casbin_rule` (`ptype`,`v0`,`v1`,`v2`,`v3`,`v4`,`v5`,`v6`,`v7`) VALUES 
empty slice found
```

### CODE REVIEW

**Comment by hsluoyz** [comment]
@tangyang9464

**Review by tangyang9464 — APPROVED** [review]
LGTM

**Comment by dirablue** [comment]
@hsluoyz
hi, are you still waiting for something before merge this?

**Comment by hsluoyz** [comment]
@dirablue plz sign CLA:

image

**Comment by dirablue** [comment]
@hsluoyz
hi I signed CLA just now. 
thanks.

**Comment by hsluoyz** [comment]
@dirablue why do you provide empty line?

**Pull request merged**
